reflex/experimental/__init__.py

```diff
@@ -42,6 +42,7 @@ def toast(self):
     asset=asset,
     client_state=ClientStateVar.create,
     hooks=hooks,
+    vars=vars,
```

Comment:
need to add `from . import vars as vars`, otherwise this name is the builtin `vars`

```python
>>> rx._x.vars
<built-in function vars>
```